canonical-web-and-design/www.canonical.com
# Reduce docker image size

## Done
Reduce size of docker image

## QA
- on master run `./run clean && ./run build`
- on `master` run `docker build --build-arg REVISION_ID=test -t c.com:master .`
- checkout this feature branch
- run `docker build -t c.com:feature .`
- run `docker images` and see the image tagged as `feature` is smaller than `master`
- run `docker run -ti -p 8002:80 c.com:feature`
- browse localhost:8002 and make sure the site works
